layouts/tanzu/single.html

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html lang="{{ .Site.Language.Lang }}">
+  <head>
+    {{ partial "head.html" . }}
+  </head>
+  <body id='{{ replaceRE "(\\s)" "" (lower .Title) }}'class="td-{{ .Kind }} td-topic">
+    {{ partial "amplitude.html" (dict "context" . "event" "tanzu viewed") }}
+    <header>
+      {{ partial "navbar.html" . }}
+    </header>
+    
+    <div class="container-fluid td-outer">
+        {{ partial "live-notify.html" . }}
+        <div class="td-main ">
+            <div id="hero" class="overflow-hidden td-box--gray-dark mt-lg-5 mt-7 py-lg-5">
+                <div class='container'>
+                    {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
+                    <div class="row align-items-center py-lg-5">
+                        <div class="col-12 col-lg-6 py-2 py-lg-5 pl-lg-0">
+                            <h1>{{ .Title }}</h1>
+                            <p class="lead">{{ .Description }}</p>
+                        </div>
+                        <div class="col-12 col-lg-6 py-2 py-lg-5 pr-lg-0 position-relative">
+                            <img class="product-icon position-absolute" alt="" src='{{ printf "%s%s%s" "/images/icons/" .Params.icon ".svg" | relURL }}'>
+                            <!-- Link Boxes -->
+                            <!-- TODO: Proper CSS fix for make link boxes the same size -->
```

Comment:
I've put a hacky fix here which seems to work, but I don't believe is 100% the correct fix. Marked with a TODO to revisit when time allows